### PR TITLE
update deps, add integration tests

### LINKED ISSUE #3: Release
Release element on Hex.

### PR DESCRIPTION
**Pull request by mat-hek**
closes #3

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   use Mix.Project
 
   @version "0.2.1"
-  @github_url "https://github.com/membraneframework/membrane-element-audiometer"
+  @github_url "https://github.com/membrane_audiometer_plugin"
 
   def project do
     [
-      app: :membrane_element_audiometer,
+      app: :membrane_audiometer_plugin,
```
Comment (bblaszkow06) [inline mix.exs:9]: Fix name and description below. You can also use the template from `membrane_template_plugin` and bump Elixir version a bit

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Membrane Multimedia Framework: AudioMeter Element
 
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane-element-audiometer.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane-element-audiometer)
+[![CircleCI](https://circleci.com/gh/membrane_audiometer_plugin)
```
Comment (bblaszkow06) [inline README.md]: This breaks the link. Also, if you plan to release it, you can add docs & hex badges.

File: test/membrane_element_audiometer/peakmeter_test.exs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    assert_pipeline_notified(pipeline, :peakmeter, :underrun)
+
+    Enum.each(data, fn payload ->
+      assert_sink_buffer(pipeline, :sink, %Membrane.Buffer{payload: received_payload})
+      assert payload == received_payload
+    end)
+
+    Testing.Pipeline.stop(pipeline)
+    assert_pipeline_playback_changed(pipeline, _, :stopped)
```
Comment (bblaszkow06) [inline test/membrane_element_audiometer/peakmeter_test.exs]: Credo will complain about `_`. You don't have to add it now, but you can run checks using config from here: https://github.com/membraneframework/membrane_rtp_plugin/blob/e32b6bc000db54dedda0ce7b08e999b24c6e51a1/.credo.exs

File: mix.exs [hunk 4]
```diff
@@ -1,12 +1,12 @@
-defmodule Membrane.Element.Audiometer.Mixfile do
+defmodule Membrane.Audiometer.Plugin.Mixfile do
   use Mix.Project
 
   @version "0.2.1"
```
Comment (bblaszkow06) [inline mix.exs]: You are breaking API, might consider version bump :wink:

**Review by bblaszkow06 — CHANGES_REQUESTED** [review]

File: test/membrane_element_audiometer/peakmeter_test.exs (hunk 3 above)
Comment (mat-hek) [reply test/membrane_element_audiometer/peakmeter_test.exs]: Thanks, for the spot, I'll add it even though I don't have tu. I'll run the checks tu ;)

File: mix.exs (hunk 1 above)
Comment (bblaszkow06) [reply mix.exs:9]: Bump bump

**Review by bblaszkow06 — APPROVED** [review]

File: mix.exs (hunk 1 above)
Comment (mat-hek) [reply mix.exs:9]: version is bumped and name and description are updated

Comment (bblaszkow06) [reply mix.exs:9]: > use the template from `membrane_template_plugin`

> and bump **Elixir** version a bit

**Pull request merged**